pkg/storaged/stratis-panel.jsx

```diff
@@ -80,17 +80,17 @@ export function stratis_rows(client) {
         return client.stratis_pools[path_a].Name.localeCompare(client.stratis_pools[path_b].Name);
     }
 
-    function cmp_locked_pool(uuid_a, uuid_b) {
+    function cmp_stopped_pool(uuid_a, uuid_b) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230425-100703-453867ac-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)